src/msg/async/Protocol.h

```diff
@@ -106,6 +106,7 @@ class Protocol {
   AsyncConnection *connection;
   AsyncMessenger *messenger;
   CephContext *cct;
+  ceph::mono_time throttle_prev = ceph::mono_clock::zero();
```

Comment:
`ceph::coarse_mono_time` is better because it will more likely avoid a system call.